gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/BaseDistributionExtension.java

```diff
@@ -133,6 +137,14 @@ public final void setProductType(ProductType productType) {
         this.productType.set(productType);
     }
 
+    public final Provider<Boolean> getLenientInternalLocks() {
```

Comment:
you can expose a `Property<Boolean>` avoid the setter and allow for lazy for configuration